src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx

```diff
@@ -29,6 +30,41 @@ export interface IProps {
     onClikLinkCb?: (tag: IOpdsBaseLinkView) => () => void | undefined;
 }
 
+const Duration = (props: {
+    duration: ITimeDuration;
+    __: I18nTyped;
+}) => {
+
+    const { duration, __ } = props;
+
+    let sentence = "";
+    sentence += duration?.days ? `${duration.days} ${__("publication.duration.days")} ` : "";
+    sentence += duration?.hours ? `${duration.hours} ${__("publication.duration.hours")} ` : "";
+    sentence += duration?.minutes ? `${duration.minutes} ${__("publication.duration.minutes")} ` : "";
+    sentence += sentence ? `${__("publication.duration.and")} ` : "";
+    sentence += duration?.seconds ? `${duration.seconds} ${__("publication.duration.seconds")}` : "";
+
+    sentence = sentence.trim();
```

Comment:
I do not have an objection against this time format, but I think we should get some feedback to check that this is an acceptable representation depending on language locale.

Another common notation for time values that do not reach days is `HH:MM:SS` (here, ignoring milliseconds, for brevity), with the advantage of being relatively locale-agnostic.
```
const str = `${hours > 0 ? (hours.toString().padStart(2, "0") + ":") : ``}${minutes > 0 ? (minutes.toString().padStart(2, "0") + ":") : `00:`}${seconds > 0 ? (seconds.toString().padStart(2, "0")) : `00`}`;
```

Reply:
Please put your algo in mind directly in the issue tracker. I'm not in your mind :)

Reply:
It's a "human-friendly" version (i.e. minutes always shown as `00` even if zero, padded string to ensure consistent double-digit `**` even if `<10`) of NPT Normal Play Time from RFC 2326:
https://www.w3.org/TR/media-frags/#naming-time
https://tools.ietf.org/html/rfc2326#page-17

Reply:
I'm totally agree with you. 

thanks you Daniel for your proposition